Cooper/DAO/GameDAO.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cooper.DAO.Models;
+using Cooper.ORM;
+using NLog;
+
+namespace Cooper.DAO
+{
+    public class GameDAO : IDAO<GameDb>
+    {
+
+        private CRUD crud;
+        Logger logger;
+
+        private string table;
+        private string idColumn;
+        private List<string> attributes;
```

Comment:
"Set" is a better structure for it. You can read about most common data structures and its usage area.